ReleaseProcessAutomation/Steps/StartReleaseStep.cs

```diff
@@ -69,30 +73,37 @@ public void Execute (string? commitHash, bool pauseForCommit = false, bool noPus
       throw new ArgumentException(message);
     }
 
-    if (_gitClient.IsOnBranch("hotfix/"))
-    {
-      _log.Debug("On branch 'hotfix', calling branch from hotfix");
-      _branchFromHotfixStep.Execute(commitHash, startReleasePhase, pauseForCommit, noPush);
-    }
-    else if (_gitClient.IsOnBranch("develop"))
-    {      
-      _log.Debug("On branch 'develop', calling branch from develop");
-      _branchFromDevelopStep.Execute(commitHash, pauseForCommit, noPush, startReleasePhase);
-    }
-    else if (_gitClient.IsOnBranch("master"))
-    {
-      _log.Debug("On branch 'master', calling branch from master");
-      _branchFromMasterStep.Execute(commitHash, startReleasePhase, pauseForCommit, noPush);
-    }
-    else if (_gitClient.IsOnBranch("release/"))
+    if (_gitClient.IsOnBranch("release/"))
     {
       _log.Debug("On branch 'release', calling branch from release");
       _branchFromReleaseStep.Execute(commitHash, pauseForCommit, noPush);
     }
     else
     {
-      const string message = "You have to be on either a 'hotfix/*' or 'release/*' or 'develop' or 'master' branch to release a version";
-      throw new InvalidOperationException(message);
+      if(!startReleasePhase)
+        _console.WriteLine(
```

Comment:
it's like a live demo / screenshot of the new behavior, please. I think the code says that the text is printed first, not last (after the options) and I'd like to verify how it looks.